frontend/src/pages/user/ModalEditUser.js

```diff
@@ -0,0 +1,146 @@
+import React, {useEffect, useState} from 'react';
+import Modal from "react-bootstrap/Modal";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+import validateUserName from "../../validation/UserValidationRules";
+import ErrorMessage from "../../messages/errorMessage";
+
+function ModalEditCustomer(props) {
+
+    const [userDto, setUser] = useState({
+        id: '',
+        name: '',
+        surname:'',
+        birthday: ''
+    });
+    const [validError, setError] = useState([]);
+    const [errorMessage, setErrors] = useState('');
+
+    const handleName = (e) => {
+        setUser(preState => ({
+            ...preState,
+            name: e.target.value
+        }));
+    };
+    const handleSurname = (e) => {
+        setUser(preState => ({
+            ...preState,
+            surname: e.target.value
+        }));
+    };
+
+    const handleBirthday = (e) => {
+        setUser(preState => ({
+            ...preState,
+            birthday: e.target.value
+        }));
+    };
+
+    const currentCustomerId = localStorage.
+    getItem("currentCustomerId") != null ? localStorage.
+    getItem("currentCustomerId"): 0;
+
+    useEffect(() => {
+        if (props.props.editShow === true) {
```

Comment:
in js props.props.editShow will be enough to evaluate it. So use if (props.props.editShow) instead